grpc/src/main/java/io/stargate/grpc/service/ExceptionHandler.java

```diff
@@ -134,6 +162,7 @@ private void handlePersistenceException(PersistenceException pe) {
   }
 
   private void handleUnavailable(UnavailableException ue) {
+    LOG.error(getErrorMessage(Status.UNAVAILABLE, ue));
```

Comment:
so here I would add the stack, I know status is UNAVAILABLE, but this is comming from the UnavailableException and we need to know what was unavailable. Actually we could print it also for the other case when UNAVAILABLE is sent bac as well, this bootsrapping situation that should actually never happen.

Reply:
final impl matrix logging stacktrace for server errors @ivansenic @tatu-at-datastax 

|  | Logged | Not Logged |
| ------------- | ------------- | ------------- |
| **Stacktrace logged**  |StatusException, StatusRuntimeException, UnhandledClientException, SERVER_ERROR, PROTOCOL_ERROR, UNPREPARED, CAS_WRITE_UNKNOWN, CONFIG_ERROR, TRUNCATE_ERROR, CDC_WRITE_FAILURE, READ_FAILURE, FUNCTION_FAILURE, WRITE_FAILURE, UNAVAILABLE, IS_BOOTSTRAPPING  | ---NA--- |
| **Stacktrace not logged**  | WRITE_TIMEOUT, READ_TIMEOUT | INVALID, SYNTAX_ERROR, BAD_CREDENTIALS, OVERLOADED, UNAUTHORIZED, ALREADY_EXISTS |